projects/plugins/jetpack/extensions/blocks/recipe/hero/edit.js

```diff
@@ -0,0 +1,17 @@
+/**
+ * External dependencies
+ */
+import { InnerBlocks } from '@wordpress/block-editor';
+
+function RecipeHeroEdit( { className, hasInnerBlocks } ) {
+	return (
+		<div className={ className }>
+			<InnerBlocks
+				allowedBlocks={ [ 'core/image', 'jetpack/slideshow' ] }
+				renderAppender={ ! hasInnerBlocks && InnerBlocks.ButtonBlockAppender }
```

Comment:
I wonder if we should have an empty slideshow or image block here, so folks see that they can add an image there. Without it, I think it's easy to miss that spot, and then you'll want to add your own images later on and will end up adding them elsewhere.